.github/workflows/build_deploy.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - '[0-9].[0-9]*'
       - '[0-9].x'
+      # special branches used to test this workflow
+      # before merging/releasing
+      - build_deploy*
```

Comment:
have we ever used the `workflow_dispatch` for actually publishing a release?

should we just repurpose the manual triggering in order to do this validation?

Reply:
this question might need to be brought to @DataDog/python-guild

Reply:
yup, agreed, we'd have to change more about the workflow in order to support using `workflow_dispatch` in that way.

no, my comment is not blocking, just a musing that that is a possible alternative approach that might be easier for people.